keel-core/src/test/kotlin/com/netflix/spinnaker/keel/actuation/EnvironmentPromotionCheckerTests.kt

```diff
@@ -88,6 +92,7 @@ internal class EnvironmentPromotionCheckerTests : JUnit5Minutests {
     )
   }
 
+  // todo eb: remove these tests (in favor of EnvironmentConstraintRunnerTests and NewEnvironmentPromotionCheckerTests)
```

Comment:
Leaving these tests here to show compatibility, but have written new tests for the new functionality. This is a todo to remove these tests.